src/patternfly/components/NotificationDrawer/examples/NotificationDrawer.md

```diff
@@ -0,0 +1,138 @@
+```
+
+### Accessibility
+| Attribute | Applied to | Outcome |
+| -- | -- | -- |
+| `aria-expanded="false"` | `.pf-c-notification-drawer__group-toggle` | Indicates that the group notification list is hidden. |
+| `aria-expanded="true"` | `.pf-c-notification-drawer__group-toggle` | Indicates that the group notification list is visible. |
+| `hidden` | `.pf-c-notification-drawer__list` | Indicates that the group notification list is hidden so that it isn't visible in the UI and isn't accessed by assistive technologies. |
+| `tabindex="0"` | `.pf-c-notification-drawer__list-item.pf-m-clickable` | Inserts the clickable list item into the tab order of the page so that it is focusable. |
+| `aria-hidden="true"` | `.pf-c-notification-drawer__group-toggle-icon > *`, `.pf-c-notification-drawer__list-item-header-icon > *` | Hides icon for assistive technologies. |
+
+### Usage
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-notification-drawer` | `<div>` | Initiates the notification drawer. |
+| `.pf-c-notification-drawer__header` | `<div>` | Initiates the notification drawer header. |
+| `.pf-c-notification-drawer__header-title` | `<h1>` | Initiates the notification drawer header title. |
+| `.pf-c-notification-drawer__header-status` | `<span>` | Initiates the notification drawer header status. |
+| `.pf-c-notification-drawer__header-action` | `<div>` | Initiates the notification drawer header action. |
+| `.pf-c-notification-drawer__body` | `<div>` | Initiates the notification drawer body. |
+| `.pf-c-notification-drawer__list` | `<ul>` | Initiates a notification list. |
+| `.pf-c-notification-drawer__list-item` | `<li>` | Initiates a notification list item. |
+| `.pf-c-notification-drawer__list-item-header` | `<div>` | Initiates a notification list item header. |
+| `.pf-c-notification-drawer__list-item-header-icon` | `<span>` | Initiates a notification list item header icon. |
+| `.pf-c-notification-drawer__list-item-header-title` | `<h2>` | Initiates a notification list item header title. |
+| `.pf-c-notification-drawer__list-item-action` | `<div>` | Initiates a notification list item action. |
+| `.pf-c-notification-drawer__list-item-description` | `<div>` | Initiates a notification list item description. |
+| `.pf-c-notification-drawer__list-item-timestamp` | `<div>` | Initiates a notification list item timestamp. |
+| `.pf-c-notification-drawer__group` | `<section>` | Initiates a notification group. |
+| `.pf-c-notification-drawer__group-toggle` | `<button>` | Initiates a notification group toggle. |
+| `.pf-c-notification-drawer__group-title` | `<div>` | Initiates a notification group toggle title. |
+| `.pf-c-notification-drawer__group-count` | `<div>` | Initiates a notification group toggle count. |
+| `.pf-c-notification-drawer__group-icon` | `<span>` | Initiates a notification group toggle icon. |
+| `.pf-m-info` | `.pf-c-notification-drawer__list-item` | Modifies a notification list item for the info state. |
```

Comment:
Is it necessary to have a modifier class on the list-item?

Reply:
Yep to indicate what kind of notification it is.